jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/QueryServiceImplTest.java

```diff
@@ -390,6 +393,30 @@ public void testGetProcessInstancesCustomWithVars() {
         processInstanceId = null;
 
     }    
+    
+    @Test
+    public void testGetProcessInstancesCustomWithoutVars() {
+
+        query = new SqlQueryDefinition("jbpmProcessSearchWithoutVars", dataSourceJNDIname);
+        query.setExpression("select p. PROCESSINSTANCEID, p.PROCESSID,  p.PROCESSNAME, p.PROCESSVERSION, " +
+                            "p.STATUS,  p.EXTERNALID, pr.STARTDATE,   p.USER_IDENTITY, p.PROCESSINSTANCEDESCRIPTION, " +
+                            "p.CORRELATIONKEY, p.PARENTPROCESSINSTANCEID, pr.LASTMODIFICATIONDATE " +
+                            "from PROCESSINSTANCELOG p " +
+                            "inner join PROCESSINSTANCEINFO pr on p.PROCESSINSTANCEID = pr.INSTANCEID");
```

Comment:
I think there are some not necessary white spaces, could we update as:
```java
query.setExpression("select p.PROCESSINSTANCEID, p.PROCESSID,  p.PROCESSNAME, p.PROCESSVERSION, " +
                    "p.STATUS, p.EXTERNALID, pr.STARTDATE, p.USER_IDENTITY, p.PROCESSINSTANCEDESCRIPTION, " +
                    "p.CORRELATIONKEY, p.PARENTPROCESSINSTANCEID, pr.LASTMODIFICATIONDATE " +
                    "from PROCESSINSTANCELOG p " +
                    "inner join PROCESSINSTANCEINFO pr on p.PROCESSINSTANCEID = pr.INSTANCEID");
```

Reply:
Done